### ISSUE TITLE
Make the "Encapsulate Field" Preview Window Bigger

### ISSUE THREAD

**Issue by CHR-IS-B** [rubberduck-vba/Rubberduck#5200]
**Justification**
The "Encapsulate Field" preview window is too small to show a 3 line Get, 3 line Let and 1 line variable declaration.  There are no scroll bars on the window so it looks like all visible text is displayed when in fact some of it is hidden

**Description**
Add scroll bars, and/or make the initial window size bigger : )

**Initial View**
-It looks like the result will only create a property-get and won't honor the setter type check boxes.
![Capture1](https://user-images.githubusercontent.com/22154950/66441369-66099080-e9eb-11e9-80a1-bf88748349c7.PNG)

**Expanded View** (after clicking the Expand/Collapse button twice)
-Ah!  Now I can see what Rubberduck is going to do and just how awesome it is!  Let's make this preview window this size by default.
![Capture2](https://user-images.githubusercontent.com/22154950/66441368-6570fa00-e9eb-11e9-8986-d2b38af84904.PNG)

**Comment by Vogel612** [comment]
This should be fixable by adjusting some size-related properties on the BindableTextEditor control included in the EncapsulateFieldView.xaml here: https://github.com/rubberduck-vba/Rubberduck/blob/d70d3cb731affaad96875a30b6dbdf9554f56dc4/Rubberduck.Core/UI/Refactorings/EncapsulateField/EncapsulateFieldView.xaml#L117-L123

The control exposes all properties that AvalonEdit's TextEditor already exposes, which includes `MinHeight` and `Height` both inherited from the `Control` class. Adding a sensible `MinHeight` should fix this issue

**Comment by retailcoder** [comment]
I agree with the resizing, and I'd say there should be scrollbars if there's more content than what's shown.

**Comment by retailcoder** [comment]
Great issue BTW, thanks for opening this!

**Comment by CHR-IS-B** [comment]
: )

**Comment by IvenBach** [comment]
This is a great 01-duckling issue. Vogel's already laid the trail of breadcrumbs out. If you have any more questions :hint: :hint: we're happy to help.
